src/app/components/pizza-bucket-page/pizza-bucket-page.component.html

```diff
@@ -0,0 +1,60 @@
+<div class="bucket-container">
+  <div class="bucket-content">
+    <div class="bucket-header">
+      <h4>Complete the order</h4>
+    </div>
+    <div
+      class="bucket-body"
+      *ngIf="(selectPizzasInBucket$ | async)?.length; else noPizzasAdded"
+    >
+      <table class="table table-strip">
+        <thead>
+          <tr>
+            <th>Name</th>
+            <th class="center-text">Qty</th>
+            <th class="center-text">Price</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr *ngFor="let pizza of selectPizzasInBucket$ | async">
```

Comment:
map your piizzas in separate selector, and assign name and price
combineLates(
selectPizzasInBucket ,
selectPizzasEntities,
)
or withLatestFrom